WcaOnRails/app/views/persons/_competitions_map.html.erb

```diff
@@ -1,17 +1,14 @@
-<% provide(:include_gmaps, true) %>
 <div id="competitions-map"></div>
 <script>
   $(function() {
-    wca.addGoogleMapsLoadedListener(function() {
-      var competitions = <%= competitions_json_for_markers @person.competitions %>
-      var map = wca.createCompetitionsMap(document.getElementById('competitions-map'));
-      wca.addCompetitionsToMap(map, competitions);
+    var competitions = <%= competitions_json_for_markers @person.competitions %>
+    var map = wca.createCompetitionsMapLeaflet('competitions-map');
+    wca.addCompetitionsToMapLeaflet(map, competitions);
 
-      $('#competitions-map').one('map-shown', function() {
-        /* Google Map is not properly initialized within a hidden container, that's how we fix it. */
```

Comment:
@jfly it seems we had that resizing issue with google maps too, but I have no idea why we didn't have it in the venue picker in the schedule editor...